File: examples/ios-host-shell/HostShell/Sources/Modules/Contacts/ContactsModule.swift
import Foundation

struct ContactsModule: ZDBModule {
    static let tables = ["contact"]

    static func bootstrap(_ driver: ZettelDriver) throws {
        _ = try driver.executeSql(sql: """
            CREATE TABLE IF NOT EXISTS contact (
                name TEXT NOT NULL,
                relationship TEXT,
                email TEXT
            )
        """)
    }
}

File: examples/android-host-shell/feature-contacts/src/main/java/com/doogat/hostshell/contacts/ContactsModule.kt
package com.doogat.hostshell.contacts

import com.doogat.hostshell.ZDBModule
import uniffi.zdb_core.ZettelDriver

object ContactsModule : ZDBModule {
    override val tables = listOf("contact")

    override fun bootstrap(driver: ZettelDriver) {
        driver.executeSql("""
            CREATE TABLE IF NOT EXISTS contact (
                name TEXT NOT NULL,
                relationship TEXT,
                email TEXT
            )
        """.trimIndent())
    }
}
